pkg/cmd/template/cmd_overlays_test.go

```diff
@@ -228,7 +227,7 @@ overlayed: true
 
 	expectedErr := "Overlaying (in following order: overlay.yml): " +
 		"Document on line overlay.yml:4: " +
-		"Expected number of matched nodes to be 1, but was 2 (lines: tpl.yml:2, tpl.yml:6)"
+		"Expected number of matched nodes to be 1, but was 2 (lines: tpl.yml:1, tpl.yml:5)"
```

Comment:
instead of changing this test. let's add one to yamlmeta/parser_test.go that explicitly pulls document's filepos and checks AsString() result to include filename.

Reply:
Done. Deliberately did not "pin-down" other constructions of `Position` to ensure the `associatedName` is properly set; seems excessive.